### PR TITLE
Fix issue #8

### PR DESCRIPTION
**Pull request by ntnmrndn**

### CODE REVIEW

File: NextGrowingTextView/NextGrowingTextView.swift [hunk 1]
```diff
@@ -165,6 +165,7 @@ open class NextGrowingTextView: UIScrollView {
 
   private func setup() {
 
+    NotificationCenter.default.addObserver(self, selector: #selector(willShowMenu(_:)), name: UIMenuController.willShowMenuNotification, object: nil)
```
Comment (muukii) [inline NextGrowingTextView/NextGrowingTextView.swift:168]: Should not we need to set an object as _textView?
I thought it's better to set limitations of scope handling notifications that come from text views.

Comment (muukii) [reply NextGrowingTextView/NextGrowingTextView.swift:168]: ah, object would be UIMenuController, so my suggestion might be wrong

Comment (ntnmrndn) [reply NextGrowingTextView/NextGrowingTextView.swift:168]: Yes, we can't use the object parameter here :(

Comment (muukii) [reply NextGrowingTextView/NextGrowingTextView.swift:168]: I understood, thanks!

**Pull request merged**